docs/administration/authentication-providers/index.md

```diff
@@ -67,6 +63,21 @@ Note that even when enabled, **this functionality is only active when there is
 
 Also, when using the Active Directory provider, this function will only be active when **allowFormsAuthenticationForDomainUsers** is set to **false**.
 
+## Automatic user creation
+
+The active directory and OpenID connect providers will, by default, automatically create a new user record for any user who can successfully authenicate but is not currently recognised (based on the checks and fallbacks described above).
+
+This has its benefits in some scenarios, for example if groups from Active Directory have been assigned accees to Teams in Octopus, then no adminstration is required in Octopus for new users who are added to those groups in Active Directoy. All the users need to do is login in to Octopus and a user will be created and it'll will be associated with the correct Team(s), based on group assignement.
+
+However, this automatic user creation doesn't suit all scenarios so as of v3.17 it can be disabled. To disabled automatic user creation for the Active Directory provider use the following command
```

Comment:
I reckon this could use a header which will end up in the TOC.

### Disabling automatic user creation